.github/workflows/code-checks.yml

```
@@ -106,3 +109,23 @@ jobs:
         with:
           token: ${{secrets.CODECOV_TOKEN}}
           files: ./app/mobile/coverage/.tmp/coverage-0.json
+
+  e2e-tests:
```

Per naming elsewhere in the file
```suggestion
  test-e2e:
```